### PR TITLE
 Adds status return to HWCPoll

### PR DESCRIPTION
**Pull request by munish-b**

### CODE REVIEW

File: common/utils/hwcutils.cpp [hunk 1]
```diff
@@ -22,15 +22,17 @@
 
 namespace hwcomposer {
 
-void HWCPoll(int fd, int timeout) {
+int HWCPoll(int fd, int timeout) {
```
Comment (kalyankondapally) [inline common/utils/hwcutils.cpp]: @munish-b Can you update commit message as per this:
https://github.com/intel/IA-Hardware-Composer/wiki/Contributions

Comment (munish-b) [reply common/utils/hwcutils.cpp]: I have updated the commit message.

File: os/android/iahwc2.cpp [hunk 2]
```diff
@@ -150,7 +150,7 @@ HWC2::Error IAHWC2::Init() {
   // Start the hwc service
   // FIXME(IAHWC-76): On Android, with userdebug on Joule this is causing
   //        to hang in a loop while cold boot before going to home screen
-  // hwcService_.Start(*this);
+  hwcService_.Start(*this);
```
Comment (kalyankondapally) [inline os/android/iahwc2.cpp]: @munish-b  You want to remove this change for  now ?

**Comment by kalyankondapally** [comment]
merged thanks

**Pull request closed without merging**